tests/views/test_collection_exercise.py

```diff
@@ -489,12 +489,12 @@ def test_collection_exercise_sample_check_count(
             "expectedTotal": 10,
             "currentTotal": 10,
         }
-
+        
         response = self.client.get(f"/surveys/{short_name}/{period}", follow_redirects=True)
 
         self.assertEqual(response.status_code, 200)
         self.assertIn("Ready for live".encode(), response.data)
-        self.assertIn("14 March 2018".encode(), response.data)
+        self.assertIn("Sample loaded: ".encode(), response.data)
```

Comment:
We aren't mocking the formatted sample in this test so it won't display in this format. Sample loaded only appears on a successful sample upload so should test the same outcome.

Reply:
Ok, that's fine, but why are we asserting against something else that is not related to the test. In fact the test isn't asserting the count at all :upside_down_face: